### ISSUE TITLE
Folder Structure

### ISSUE THREAD

**Issue by Sureiya** [slashiee/cemu_graphic_packs#49]
I know the verbosely named folders are nice to have, but I feel like it would be way more useful to have single folder `graphicPacks`. 

Instead the graphics pack category could be in the actual pack folder name, in the graphicPack name field, or categorized in a table in the docs.

It would be a lot simpler and easier to update if we could just drag the whole folder in, or for those of us comfortable with git, just do a git pull.

If no one agrees, I'll work on a PR build script or something to move them.

**Comment by slashiee** [comment]
This could probably be addressed, with wiki pages telling what packs are part of what. I don't know how @Darkemaste or @getdls think about this, though.

**Comment by slashiee** [comment]
I just created the Wiki part, but I don't know about the other's opinions yet. https://github.com/slashiee/cemu_graphic_packs/wiki

**Comment by slashiee** [comment]
I've created a PR that would suit your needs at #50, but I'm still waiting on the others before doing anything else related to this.

**Comment by getdls** [comment]
I think it's an improvement, and it doesn't require any more work from our side. That and future texture packs can probably be a flat "main folder" structure.

Enhancements,  I'd say the majority want better shadows that so keeping those at x2  and sticking them in the same folder is a reasonable compromise. It's less work to un-chek the box than to find it on git and copy .

Workarounds, just give them a really descriptive name and stick them  in the main folder, from the number of "AMD is broken questions" ppl don't know where to look for them. 

I'm still not sold on low res packs, has anyone with a slow card benched marked them against no gfx pack active at all? But I'm not that annoyed so do as you please there.  

I'd keep fun and AA removal in a separate folder, too many questions on reddit why they should be used.

Any other suggestions?

**Comment by slashiee** [comment]
@getdls I've made several pack names more descriptive according to what you have said. 8a2761e9ecf786df257693cc4c69914fee1e4384

**Comment by slashiee** [comment]
I've removed 720p improved packs and Fun packs altogether. I'm unsure about Performance packs though

**Comment by Darkemaste** [comment]
I don't really think this is needed tbh, I think even if we did this people would then complain about the graphic pack window in Cemu being too cluttered. (Currently if you use a single category you only see 3-4 resolutions for each supported title in Cemu's graphic packs window, if you combined them all you'd then see 10-12 resolutions for each supported title instead.)

I don't really see it hard to update any particular folder/category if that is the concern, the entire repo is a tiny download and you only need to copy the contents of a single category/folder to the graphic packs directory in Cemu to update said category/repo anyway.

Also don't really think it would be a good idea to move workaround shader edits to the main packs either, I don't think Nvidia users would of wanted a handful of enemies missing eyes in BotW with the AMD remove_eyes workaround for AMD users in 1.7.3, or Nvidia owners and non-Polaris/GCN4 AMD GPU owners to have rules removed/altered for Polaris/GNC4-specific bug that doesn't effect them. It might also makes shader edits that need different values for both Nvidia/AMD (such as Xeno) more of a hassle. (the latter is also worth noting in case a shader edit for Smash specifically for Nvidia is needed in 1.7.6)

**Comment by slashiee** [comment]
@Sureiya I guess since there hasn't been any responses since and @Darkemaste disagrees, a script to move  them automatically is the option at this point.

**Comment by slashiee** [comment]
@Sureiya AppVeyor does the job nicely.
https://ci.appveyor.com/api/buildjobs/pp5wvxvq4bocigo3/artifacts/graphic_packs_v2-1.zip
